firefox-ios/Client/Application/WindowManager+DebugUtilities.swift

```diff
@@ -35,12 +37,15 @@ extension WindowManagerImplementation {
 
         // Note: this is provided as a convenience for internal debugging. See `DefaultTabDataStore.swift`.
         for (idx, uuid) in tabDataStore.fetchWindowDataUUIDs().enumerated() {
-            result.append("    \(idx + 1): Window \(short(uuid))\n")
             let baseURL = fileManager.windowDataDirectory(isBackup: false)!
             let dataURL = baseURL.appendingPathComponent("window-" + uuid.uuidString)
+            if idx == 0 {
```

Comment:
I think this check is unnecessary and only complicates the loop iteration. I suggest putting the code before or after the loop

`result.insert(object, at: 0)`